src/kontrol/__main__.py

```diff
@@ -156,7 +166,7 @@ def _compare_versions(ver1: KVersion, ver2: KVersion) -> bool:
 def exec_load_state(options: LoadStateOptions) -> None:
     foundry_state_load(
         options=options,
-        foundry=_load_foundry(options.foundry_root),
+        foundry=_load_foundry(options.foundry_root, add_enum_constraints=not options.enum_constraints),
```

Comment:
Should `not` be removed now that the option is inverted? Or is it intentional? Same for other `_load_foundry` calls below.

```suggestion
        foundry=_load_foundry(options.foundry_root, add_enum_constraints=options.enum_constraints),
```